### ISSUE TITLE
Task: Manual curation

### ISSUE THREAD

**Issue by ubgg** [Steem-FOSSbot/steem-fossbot-voter#23]
Add a section in the voter to manually add links for the bot to vote on.

Make it so that you can queue up several posts up for the bot to cycle trough them.

**Comment by thrize** [comment]
Good idea! This would be especially useful for voting on new posts which are under the MIN_POST_AGE_TO_CONSIDER.

For example, say you see a great post, posted 2 minutes ago, but you want to vote for it after 15 minutes in order to keep a larger part of the curation reward. You add it to the list and then after 15 minutes, on the next bot run it will be voted for.

Also the queue idea is great.

**Comment by thrize** [comment]
**Accepted this suggestion, will be implemented**
